src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -329,4 +340,7 @@ export default {
 	}
 }
 
+a.user-bubble__content {
```

Comment:
`a` already has `cursor: pointer;` applied by default, no?

Reply:
There are two rules (from server) applied, with different value. But maybe it's redundant now, since we don't redefine cursor for div